loader/src/smart-scheduling-links.js

```diff
@@ -120,7 +120,10 @@ class SmartSchedulingLinksApi {
         ...this.httpOptions,
         url: this.url,
       }).json();
+      // FIXME: We should validate the schema of the response.
```

Comment:
I’m going to add some `assertSchema()` calls tomorrow or later this week, since this highlights the fact that we should have made sure we were alerted that the data was malformed instead of processing it incorrectly.